impl/src/main/java/com/okta/sdk/impl/resource/DefaultPolicyRuleBuilder.java

```diff
@@ -25,20 +24,12 @@
 
 public class DefaultPolicyRuleBuilder<T extends PolicyRuleBuilder> implements PolicyRuleBuilder<T> {
 
-    protected String id;
     protected Integer priority;
     protected PolicyRule.TypeEnum type;
     protected PolicyRule.StatusEnum status;
-    protected Boolean isActive = true;
```

Comment:
removed?
(sorry, i'm guessing this might have already been discussed)

Reply:
yes, this is an unused field in builder. So, removed it.